### ISSUE TITLE
UMD default export breaks isomorphic React applications

### ISSUE THREAD

**Issue by jwineman** [hellosign/hellosign-embedded#129]
Server frameworks like [NextJS](https://github.com/zeit/next.js/) render the same code on the server that they do on the client to simplify the implementation of functionality like server side rendering.  If I try to import HelloSign like this:
```
import HelloSignEmbedded from 'hellosign-embedded';
```
I get the following error during the server build:
```
ReferenceError: window is not defined
    at Object.<anonymous> (/front-end/node_modules/hellosign-embedded/umd/embedded.development.js:10:4)
```
To fix I have to import like this:
```
const HelloSignEmbedded = typeof window !== 'undefined' ? require('hellosign-embedded') : null;
```
Ideally the `import` wouldn't have any side-effects so I can include it normally and only bind to `window` when I `new` up an instance of the class.

**Comment by nathanbuchar** [comment]
This should now be fixed in `hellosign-embedded@v2.7.2`. Thanks for your patience!

**Comment by jwineman** [comment]
thanks!
